### PR TITLE
Pr/table preparations

### PR DESCRIPTION
**Pull request by MethodenMann**
Various independent commits that are needed for the new table

### CODE REVIEW

File: packages/tocco-ui/src/FormattedValue/typeFormatters/BooleanFormatter.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+    style={{fontSize: `${props.theme.fontSize.base}rem`}}
+    icon={props.value ? 'check' : 'times'}
+  />
 
 BooleanFormatter.propTypes = {
-  value: PropTypes.bool
+  value: PropTypes.bool,
+  theme: PropTypes.shape({
+    fontSize: PropTypes.shape({
+      base: PropTypes.number
```
Comment (rzueger) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/BooleanFormatter.js]: (theme, fontSize and base all are required apparently)

File: packages/tocco-ui/src/FormattedValue/typeFormatters/MultiSelectFormatter.js [hunk 2]
```diff
@@ -9,7 +9,10 @@ const MultiSelectFormatter = ({value, options, breakWords}) => {
   return <Typography.Span breakWords={breakWords}>
     {value && value.length > 0
       ? value
-        .map(v => options && options.linkFactory ? options.linkFactory(v.key, v.display) : v.display)
+        .map(v => options && options.linkFactory
+          ? <span onClick={e => e.stopPropagation()}>{ options.linkFactory(v.key, v.display)}</span>
```
Comment (rzueger) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/MultiSelectFormatter.js]: (space before `options.linkFactory(...`)

File: packages/tocco-ui/src/FormattedValue/typeFormatters/SingleSelectFormatter.js [hunk 3]
```diff
@@ -8,7 +8,9 @@ const SingleSelectFormatter = ({value, options, breakWords}) => {
   value = js.getOrFirst(value)
   const display = <Typography.Span breakWords={breakWords}>{value.display}</Typography.Span>
 
-  return options && options.linkFactory ? options.linkFactory(value.key, display) : display
+  return options && options.linkFactory
+    ? <span onClick={e => e.stopPropagation()}>{ options.linkFactory(value.key, display)}</span>
```
Comment (rzueger) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/SingleSelectFormatter.js]: (space before `options.linkFactory(...`)

File: server/index.html [hunk 4]
```diff
@@ -11,7 +11,16 @@
   <script src="/static/node_modules/intl/locale-data/jsonp/de-DE.js"></script>
   <script src="/static/node_modules/intl/locale-data/jsonp/en.js"></script>
   <style>
-    html {font-size: 10px;}
+    html, body {
+      font-size: 10px;
+      height: 100vh;
+      width: 100vw;
+      max-width:100%;
```
Comment (rzueger) [inline server/index.html]: (space)

File: packages/tocco-ui/src/FormattedValue/typeFormatters/MultiSelectFormatter.js (hunk 2 above)
Comment (MethodenMann) [reply packages/tocco-ui/src/FormattedValue/typeFormatters/MultiSelectFormatter.js]: human linter strikes back 😄

**Pull request merged**